great-idea-website/index.html

```diff
@@ -15,49 +15,70 @@
 </head>
 
   <body>
+  <nav class="navigation">
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+  </nav>
+  
+<header class="header">
+  <img class="idea-img" src="img/logo.png" alt="Great Idea! Company logo.">
+  <h1 class="innovation-header">Innovation</br>On</br>Demand</h1>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
+  <h2 class="getstarted-button">Get Started</h2>
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  <img class="circle-img" src="img/header-img.png" alt="Image of a code snippet.">
+</header>
 
+ <section class="top-section">
```

Comment:
You did a really great job with class names throughout the html. They are clear and descriptive of the relative position & content.